src/main/java/duke/Event.java

```diff
@@ -0,0 +1,50 @@
+package duke;
+public class Event extends Task{
+    private String eventTime;
+
+    /**
+     * Constructor for a Event object (Only takes in content as event time is included in content).
+     *
+     * @param taskContent Content to be stored in the Event object.
+     */
+    public Event(String taskContent) {
+        super(taskContent.split(" /at ")[0], "E");
+        eventTime = Parser.parseTiming(taskContent.split(" /at ")[1]);
```

Comment:
Maybe Parser.parseTiming(taskContent.split(" /at ") could be an array variable. Would that make the code more readable? 🤔